src/lib.rs

```diff
@@ -133,6 +133,7 @@ pub mod test;
 
 #[cfg(test)]
 mod tests {
+    #[cfg(feature = "rsa_signing")]
```

Comment:
A few of the tests depend on BN_rand_range, which in turn pulls in some other stuff. Options:
- Ship the actual BN_rand_range even without RSA signing;
- Move that subset of the BN tests to their own file and include it conditionally;
- Some other way to disable these tests depending on the Rust feature config.
Your preference / thoughts?